lang/LangSource/SC_LanguageConfig.hpp

```diff
@@ -65,6 +66,11 @@ class SC_LanguageConfig
 
 	bool forEachIncludedDirectory(bool (*)(const Path&)) const;
 
+	bool getExcludeDefaultPaths() const;
+	void setExcludeDefaultPaths(bool value);
```

Comment:
these can be defined inline, like `get/setPostInlineWarnings` below, to save space